controller/transfer/cognition/minioUpload.py

```diff
@@ -0,0 +1,37 @@
+from typing import List
+from submodules.model.cognition_objects import project as cognition_project
+from submodules.model.cognition_objects import conversation
+from submodules.model.enums import TaskType
+from controller.task_queue import manager as task_queue_manager
+
+
+def handle_cognition_file_upload(path_parts: List[str]):
+
+    if path_parts[1] != "_cognition":
+        return
+
+    if path_parts[3] == "chat_tmp_files":
+        if path_parts[5] == "queued":
```

Comment:
What do you think of combining the conditions as we're not distinguishing between both states further?